config/prow/update-jobs.yaml

```diff
@@ -28,4 +28,24 @@ roleRef:
 subjects:
 - apiGroup: rbac.authorization.k8s.io
   kind: Group
-  name: system:nodes
\ No newline at end of file
+  name: system:nodes
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: update-jobs-sa
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
```

Comment:
Should this Role instead of ClusterRole since we making it namespace scoped?

Reply:
Yeah, Role to enable write configmaps on prow namespace (in our case default), with service account in test-pods